client/src/components/DatasetInformation/DatasetError.vue

```diff
@@ -116,9 +117,9 @@ export default {
     data() {
         return {
             message: null,
-            email: null,
             errorMessage: null,
             resultMessages: [],
+            emailTitle: this.l("Your email address"),
```

Comment:
`this.l()` is a duplication of `emailTitle | l` used in the template above. I suggest to instead write:
`<span class="mr-2 font-weight-bold v-localize">{{ emailTitle }}</span>`, moving forward.

Reply:
Agreed - updated.